js/packages/teams-ai/src/Application.ts

```diff
@@ -747,6 +747,111 @@ export class Application<TState extends TurnState = DefaultTurnState> {
     }
 }
 
+/**
+ * A builder class for simplifying the creation of an Application instance.
+ * @template TState Optional. Type of the turn state. This allows for strongly typed access to the turn state.
+ */
+export class ApplicationBuilder<TState extends TurnState = DefaultTurnState> {
+    private _options: Partial<ApplicationOptions<TState>> = {};
+
+    /**
+     * Configures the application to use long running messages.
+     * @param {BotAdapter} adapter The adapter to use for routing incoming requests.
+     * @param {string} botAppId The Microsoft App ID for the bot.
+     * @returns {this} The ApplicationBuilder instance.
+     */
+    public withLongRunningMessages(adapter: BotAdapter, botAppId: string): this {
+        if (!botAppId) {
+            throw new Error(
+                `The Application.longRunningMessages property is unavailable because no botAppId was configured.`
```

Comment:
Something along the lines of "botAppId cannot be null or undefined" might be more apt.